docs/naps/6-contributable-menus.md

```diff
@@ -318,13 +328,38 @@ Plugins
 │  ├─ predict_from_model
 ```
 
+### Item Descriptions
+
+To allow users to browse menus more effectively without having to click each
+item to figure out what it does, menu contributions should be given a `description`
+field that is available when the user is inspecting the menu. The `description`
+should be required, so that 
```

Comment:
To be finished.
Also just to clarify we would want description to be 'tooltip' field in `Action` which appears in the bottom status bar right?

Reply:
Done.